### PR TITLE
E2e ginkgo

### PR DESCRIPTION
**Pull request by enxebre**
Run e2e suite based on https://github.com/openshift/cluster-api-actuator-pkg/pull/29
See also:
https://github.com/openshift/cluster-api-provider-aws/pull/170
https://github.com/openshift/kubernetes-autoscaler/pull/43

---
bottom line: How to do it:

By copying the latest /pkg/e2e/e2e_test.go file from github.com/openshift/cluster-api-actuator-pkg
repo under /test/e2e_test.go and running `dep ensure` command.
The file /test/e2e_test.go is not included in the payload and is purely
to pull all deps needed by the test suite.
This is the expected process of pulling the latest changes in the test suite.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -52,8 +52,13 @@ build-integration: ## Build integration test binary
...
 
-test-e2e:
-	go run ./vendor/github.com/openshift/cluster-api-actuator-pkg/pkg/e2e/openshift/*.go -alsologtostderr
+test-e2e: ## Run openshift specific e2e test
+	go test -timeout 60m \
+		-v ./vendor/github.com/openshift/cluster-api-actuator-pkg/pkg/e2e \
+		-kubeconfig $${KUBECONFIG:-~/.kube/config} \
+		-machine-api-namespace $${NAMESPACE:-openshift-machine-api} \
+		-ginkgo.v \
+		-args -v 5 -logtostderr true
```
Comment (frobware) [inline Makefile]: What's the trailing `true` for?

Comment (bison) [reply Makefile]: It's the value for the `-logtostderr` argument.

Comment (frobware) [reply Makefile]: Isn't that implicitly `true` if specified?

**Comment by ingvagabund** [comment]
@enxebre can you update commit message of the `Revendor for cluster-api-actuator-pkg ginkgo suite` commit with something similar to:

```
By copying the latest /pkg/e2e/e2e_test.go file from github.com/openshift/cluster-api-actuator-pkg
repo under /test/e2e/e2e_test.go and running `dep ensure` command.
The file /test/e2e/e2e_test.go is not included in the payload and is purely
to pull all deps needed by the test suite.
This is the expected process of pulling the latest changes in the test suite.
```

?

**Pull request merged**